packages/data-mate/src/column/utils.ts

```diff
@@ -40,9 +41,12 @@ export function mapVectorEachValue<T, R = T>(
     builder: Builder<R>,
     fn: (value: T, index: number) => Maybe<R>,
 ): Vector<R> {
+    const containsArray = vector.type === VectorType.Tuple || vector.config.array
+        || vector.type === VectorType.Any;
+
     function _mapValue(value: T|readonly Maybe<T>[], index: number): Maybe<R>|readonly Maybe<R>[] {
-        if (isArrayLike<readonly Maybe<T>[]>(value)) {
-            return value.map((v): Maybe<R> => (
+        if (containsArray && value != null) {
```

Comment:
To support the `Any` we need this logic, since `Any` doesn't have to be any array
```suggestion
        if (containsArray && Array.isArray(value)) {
```